### PR TITLE
Add sc playground subsection

### PR DESCRIPTION
**Pull request by Thykof**
Link to the issue: https://github.com/massalabs/massa-sc-playground/issues/19

The playground is not deployed yet, we will merge this PR when it is deployed and running.

### CODE REVIEW

**Comment by longgamma-massa** [comment]
@Thykof update the documentation section of the Playground to:

- title: 
Smart-contract Playground

- description (body):
[Smart-contract Playground](massa.net/testnet/sc-playground) is a simple interface for writing, unit-testing, compiling, and sharing the smart-contract code written in Assembly Script. 
Make sure you check the available import functions in the example code provided in the code-space. Some functions might not yet be available. 

P.S. I linked the URL for the Playground to the first phrase in the description, instead of 'via this link'. Pls make sure it works once we publish the interface to the URL.

**Comment by Thykof** [comment]
Here is the result: 
image

**Comment by Thykof** [comment]
Here is the result:
image

File: web3-dev/smart-contracts/playground.rst [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+Smart-contract Playground
+=========================
+
+`Smart-contract Playground <https://massa.net/sc-playground>`_ is a simple interface for writing, unit-testing, compiling, and sharing the smart-contract code written in Assembly Script. 
+
+Smart-contract Playground runs in your browser's local storage.
+
+At the moment, Playground is limited in terms of imported smart-contract methods.
+The available methods are:
```
Comment (qdrn) [inline web3-dev/smart-contracts/playground.rst:11]: Methods are not formatted well. See https://massa--2.org.readthedocs.build/en/2/web3-dev/smart-contracts/playground.html

**Comment by qdrn** [comment]
@gregLibert would probably be a better reviewer than me for such a documentation.

File: web3-dev/smart-contracts/playground.rst (hunk 1 above)
Comment (Thykof) [reply web3-dev/smart-contracts/playground.rst:12]: I've just pushed the fix

Comment (Thykof) [reply web3-dev/smart-contracts/playground.rst:12]: image

**Comment by qdrn** [comment]
I feel the content is a bit dry but we can start with that and improve over time.

**Review by qdrn — APPROVED** [review]

**Pull request merged**